www/widgets/DemoExampleMaker.js

```diff
@@ -134,6 +147,21 @@ class DemoExampleMaker extends Widget {
         <textarea name="dem-url"></textarea>
       </div>
     `
+
+    this._text = new Netitor ({
```

Comment:
this is throwing a linting error b/c we need to let the linter know that `Netitor` is a global variable, add it to the global variable list in the comment on line 1